client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -60,27 +59,24 @@ class UsedOnTable extends PureComponent {
   }
 
   renderRow(data, index) {
```

Comment:
I understand refactoring this is not part of this job, but I would have split this into it's own component which owuld have made it easier to test.